src/BlockEntities/EnderChestEntity.cpp

```diff
@@ -75,7 +75,7 @@ void cEnderChestEntity::OpenNewWindow()
 void cEnderChestEntity::LoadFromJson(const Json::Value & a_Value, cItemGrid & a_Grid)
 {
 	int SlotIdx = 0;
-	for (Json::Value::iterator itr = a_Value.begin(); itr != a_Value.end(); ++itr)
+	for (Json::Value::const_iterator itr = a_Value.begin(); itr != a_Value.end(); ++itr)
```

Comment:
If you're going to touch the loop, try to convert it to a c++11 range loop where possible.